README.md

```diff
@@ -20,6 +21,13 @@ Where
 - `product` is a browser name with an option of [product spec](https://github.com/web-platform-tests/wpt.fyi/blob/master/api/README.md)
   - `{browser-name}[-{browser-version}[-{os}[-{os-version}]]]`
   - e.g. `chrome`, `safari-12`, or `firefox-63.0-linux`
+  - when `product` is omitted, this `link` applies to tests across all browsers
+- `URL` is a bug URL; `URL` is non-optional unless a `label` is present.
+- `label` is a label to its tests; `label` is used at a test-level and do not
+  apply to subtests. When `label` is present in a link:
+  - `product`should be omitted
+  - `URL` is optional
+  - `Subtest name` is omitted
 - `Test name` is a filename, which is relative to the current directory. If it
```

Comment:
This should be `test` I think. (Existing issue.)

Reply:
```suggestion
- `test` is a filename, which is relative to the current directory. If it
```